contexts/security-v3-unstable.jsonld

```diff
@@ -1,38 +1,92 @@
 {
-    "@context": [
-        {
-            "@version": 1.1
-        },
-        "https://w3id.org/security/v2",
-        {
-            "JsonWebSignature2020": {
-                "@id": "sec:JsonWebSignature2020"
-            },
-            "publicKeyJwk": {
-              "@id": "sec:publicKeyJwk",
-              "@type": "@json"
-            },
-            "ethereumAddress": {
-                "@id": "sec:ethereumAddress"
-            },
-            "publicKeyHex": {
-                "@id": "sec:publicKeyHex"
-            },
-            "blockchainAccountId":{
-                "@id": "sec:blockchainAccountId"
-            },
-            "Bls12381G1Key2020":{
-                "@id": "sec:Bls12381G1Key2020"
-            },
-            "Bls12381G2Key2020":{
-                "@id": "sec:Bls12381G2Key2020"
-            },
-            "BbsBlsSignature2020":{
-                "@id": "sec:BbsBlsSignature2020"
-            },
-            "BbsBlsSignatureProof2020":{
-                "@id": "sec:BbsBlsSignatureProof2020"
+  "@context": [
+    {
+      "@version": 1.1
+    },
+    "https://w3id.org/security/v2",
+    {
+      "JsonWebKey2020": {
+        "@id": "https://w3id.org/security#JsonWebKey2020"
+      },
+      "JsonWebSignature2020": {
+        "@id": "https://w3id.org/security#JsonWebSignature2020"
+      },
+      "Ed25519VerificationKey2020": {
+        "@id": "https://w3id.org/security#Ed25519VerificationKey2020"
+      },
+      "Ed25519Signature2020": {
+        "@id": "https://w3id.org/security#Ed25519Signature2020",
+        "@context": {
+          "@version": 1.1,
+          "@protected": true,
+          "id": "@id",
+          "type": "@type",
+          "challenge": "https://w3id.org/security#challenge",
+          "created": {
+            "@id": "http://purl.org/dc/terms/created",
+            "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+          },
+          "domain": "https://w3id.org/security#domain",
+          "expires": {
+            "@id": "https://w3id.org/security#expiration",
+            "@type": "http://www.w3.org/2001/XMLSchema#dateTime"
+          },
+          "nonce": "https://w3id.org/security#nonce",
+          "proofPurpose": {
+            "@id": "https://w3id.org/security#proofPurpose",
+            "@type": "@vocab",
+            "@context": {
+              "@version": 1.1,
```

Comment:
Just as a syntax note @dlongley @msporny do we need to duplicate this in nested context definitions or does what is defined at the root of the context propergate?